CsvParser/csvParser.go

```diff
@@ -113,19 +111,15 @@ func makeJsonObjFile(arr []incident.Incident) *os.File {
 		return f
 	}
 
+	// Use data retrieved through the Incident interface to create incidents formatted for JSON
 	for i := 0; i < numObjects; i++ {
-		gpStart, gpEnd := arr[i].GetGoodPeriod()
-		bpStart, bpEnd := arr[i].GetBadPeriod()
-		gMetric := arr[i].GetGoodMetric()
-		bMetric := arr[i].GetBadMetric()
-		severity := arr[i].GetSeverity()
-		testsAffected := arr[i].GetTestsAffected()
-		gpInfo := arr[i].GetGoodPeriodInfo()
-		bpInfo := arr[i].GetBadPeriodInfo()
-		iInfo := arr[i].GetIncidentInfo()
-		inc := incident.IncidentData{gpStart, gpEnd, bpStart, bpEnd, gMetric, bMetric, severity, testsAffected, gpInfo, bpInfo, iInfo}
-		objs[i] = inc
+		inc := arr[i]
+		var incidentData = incident.IncidentData{}
+		incidentData.MakeJsonIncident(inc.GetIncidentData())
+		objs[i] = incidentData
 	}
+
+	// Write the incident JSON file
```

Comment:
Good work so far but do we need this function? I don't think we need it anymore (past demo days)

Reply:
Since this PR does not include the file hierarchy changes and there is no code to replace this function, this should still be included.